wiki/addnodes.md

```diff
@@ -15,126 +15,98 @@ will get more chances to find other nodes
 You can add these into your config file with `addnode=ADDNODE`. 
 See the [config file](config-file "wikilink") page about how to change your config
 
+**Make sure you have at least a few addnodes in your config or you won't be able to connect to the network.**
+
 You can also add the node with the command: `addnode <node> <add|remove|onetry>` 
 in the debug console.
 
 
-
 ## List Of Addnodes
 
-Tables with a list of addnodes, their domain name, and their region. See [cycy's hourly updating list](https://addnode.cycy.me)
-if you want another source of addnodes
+Tables with a list of addnodes, and their region. 
+
+The status on this wiki page is not updated frequently.
 
 
 ### Mainnet
 
-List of addnodes for the main network. This is the one you most likely care about
+List of addnodes for the main network. These are the ones you most likely care about.
+
+See [cycy's hourly updating list](https://addnode.cycy.me) for a more up to date status.
 
 ------------
 
-#### Online (connected within the last 24 hours)
+#### Default
+These are included in the default config. 
```

Comment:
May be good to mention that these addnodes will not be connected to if you leave them out of the config
```suggestion
These are included in the default config, but if you leave them out of the config, it will not connect to these addnodes 
```

Reply:
@nathanielcwm not sure if you saw this, but left this comment a few days ago